### PR TITLE
feat(WEBRTC-1608): add notification icon when receiving new messages in the chat

### PR DESCRIPTION
**Pull request by DeividVeloso**
- add notification icon when receiving new messages in the chat

image

### CODE REVIEW

**Comment by arghav** [comment]
@DeividVeloso could you please fix the merge conflict?

**Review by muriware — APPROVED** [review]
Functional review; it LGTM!

File: pages/rooms/index.tsx [hunk 1]
```diff
@@ -67,6 +68,8 @@ export default function Rooms({ id }: { id: string }) {
     string | undefined
   >();
 
+  const unReadMessages = useRef<TelnyxRoom['messages'] | null>(null);
```
Comment (arghav) [inline pages/rooms/index.tsx:71]:
```suggestion
  const unreadMessages = useRef<TelnyxRoom['messages'] | null>(null);
```

File: pages/rooms/index.tsx [hunk 2]
```diff
@@ -175,6 +178,7 @@ export default function Rooms({ id }: { id: string }) {
           localTracks,
           setLocalTracks,
           sendNotification,
+          unReadMessages,
```
Comment (arghav) [inline pages/rooms/index.tsx:181]:
```suggestion
          unreadMessages,
```

**Review by arghav — CHANGES_REQUESTED** [review]

**Review by muriware — APPROVED** [review]
LGTM! I left a suggestion.

File: components/RoomControls.tsx [hunk 3]
```diff
@@ -410,6 +426,37 @@ export default function RoomControls({
...
+    if (messages && messages.length > 0) {
+      const lastMessage = messages.length - 1;
+      const isNotLocalParticipantMessage =
+        messages[lastMessage].from !== localParticipant.id;
+      const existUnreadMessages = hasUnreadMessages();
+
+      if (isNotLocalParticipantMessage && !showChatBox && existUnreadMessages) {
+        return true;
+      }
+      return false;
```
[the remark this answers is not in the document]
Comment (DeividVeloso) [reply components/RoomControls.tsx:446]: Sure!!

**Pull request merged**